mod-di-converter-storage-server/src/main/resources/templates/db_scripts/associations-migration/init_wrappers.sql

```diff
@@ -280,3 +287,17 @@ $$
     RAISE NOTICE 'PROFILES_MIGRATION:: updated action_to_action_profiles';
   END
 $$;
+
+/*
+ System table for saving migration history.
+ */
+insert into metadata_internal(id, jsonb, creation_date)
+  values (public.uuid_generate_v4(), '{"name": "Migration of profiles to the use of wrappers"}', now()::timestamptz);
+
+/*
+  To check the results of the migration. Could be remove after migration.
+ */
+drop table if exists snapshots_new;
+create table snapshots_new as
```

Comment:
I think this is still using the `get_profile_snapshot` function is still using the old version and not one that uses wrapperId. So it would contain the same information as snapshot_old.

Reply:
Thanks, @okolawole-ebsco, I moved the creation of snapshot_new into the latest script.